backend/vcs/mttdoc/conversation.go

```diff
@@ -0,0 +1,27 @@
+package mttdoc
+
+import (
+	"mintter/backend/vcs"
+	"mintter/backend/vcs/hlc"
+
+	"github.com/ipfs/go-cid"
+)
+
+// NewConversationPermanode creates a new Permanode for the Mintter Conversation object.
+func NewConversationPermanode(owner cid.Cid, at hlc.Time) vcs.Permanode {
```

Comment:
Why do we need a permanode for conversations? aren't they linked to a document that already has a permanode itself?

Reply:
maybe you missed this, @burdiyan

Reply:
@juligasa I actually did respond, and pretty extensively. Don't know why it didn't end up being posted :(

Conversations are separate objects, kinda like separate documents. They have independent "timeline" and can evolve independently from the original document they point to.

Currently, all Mintter Objects need to have a Permanode, hence Conversations have them as well.

So, conversation handling is similar to backlinks from other documents.